config.yaml

```diff
@@ -1,3 +1,329 @@
 manager:
+  # The manager's private IP address. This is the address which will be used by the
+  # application hosts to connect to the Manager's fileserver and message broker.
   private_ip: ''
+
+  # The public IP of the manager to which the CLI will connect.
   public_ip: ''
+
+  ######################################################################################
+  #
+  # As long as you have set the config entries above, you can install the manager with:
+  # cfy_manager install
+  # The rest of the configuration is only required if you need to change the defaults.
+  #
+  ######################################################################################
+
+  # This is set by the installer build script. Changing this will not change the edition
+  # you are using, but may cause interesting problems.
+  premium_edition: set_by_installer_builder
```

Comment:
Let's be more explicit about what this does. Shall we?
Or better yet, why bother with having it here at all? Let's just add it to the file at build time, and be done with it.

Reply:
sed is easier to use with the sort of nested data than other forms of file injection unless we ingest the file using pyyaml and then excrete it with the modifications.

Given that we're currently doing this as a bash script this seemed like a much nicer way to deal with it.

Reply:
My problem is that this is living in a file that specifically should be handled by users, but we're saying "don't edit this line, or else...". This seems very awkward to me.

Reply:
The file that should be edited by users comes out of the other side of the create_rpm.sh script. That means that the user will see a <true|false>. This file is basically a template for that.